Performance for Serverless on Python

ajjindal opened the pull request:
- Supports Tracing
- Captures function payload with the `send-default-pii`

Commit:
EoF fix

dcramer committed:
feat: Simplify Laravel instructions (#2388)

PeloWriter commented on src/platforms/python/guides/aws-lambda/index.mdx:
```diff
@@ -52,19 +61,12 @@ sentry_sdk.init(
 
-- Automatically report all exceptions from your Lambda Functions
+- Automatically report all exceptions and transactions from your Lambda Functions
+- You can control transactions sample rate using `traces_sample_rate`. You can learn more about transactions in [Distributed Tracing](/performance-monitoring/distributed-tracing/#traces-transactions-and-spans)
```
Note - the content for this setting is forthcoming in PR 2376.

untitaker replied:
ok let's not merge this PR before that one then. It seems that even with the current version of #2376 this will 404, so we'll have to re-review.

ajjindal replied:
I took in the latest and the link still looks valid. @PeloWriter do you want me to update anything specific?

PeloWriter replied:
This link may be more helpful. So perhaps edit to:

- You can control the the representative sample of transactions you send to Sentry by [filtering transaction event](/platforms/javascript/configuration/filtering/#filtering-transaction-events)

PeloWriter commented on src/platforms/python/guides/gcp-functions/index.mdx:
```diff
@@ -25,11 +25,13 @@ from sentry_sdk.integrations.gcp import GcpIntegration
 sentry_sdk.init(
     dsn="___PUBLIC_DSN___",
     integrations=[GcpIntegration()],
+    traces_sample_rate=1.0, # adjust the sample rate in production as needed
```
We're moving to this language:

```suggestion
    traces_sample_rate=1.0, # We recommend adjusting this in production
});
```

PeloWriter reviewed (approved):
LGTM

dcramer committed:
feat: Switch to fixed navs with scrolling (#2387)

ajjindal committed:
Update src/platforms/python/guides/aws-lambda/index.mdx

Co-authored-by: Fiona <61481573+PeloWriter@users.noreply.github.com>

ajjindal committed:
Update src/platforms/python/guides/gcp-functions/index.mdx

Co-authored-by: Fiona <61481573+PeloWriter@users.noreply.github.com>

dcramer committed:
Revert "feat: Switch to fixed navs with scrolling (#2387)"

This reverts commit a3a507cb7846df70c5d677b510af7d4230687548.

MimiDumpling committed:
updated links to route correctly (#2390)

Docs has many broken/dead fragment URLs because an anchored section of a page moved or was renamed.

stevenplewis committed:
bumping hero up b/c of what I think is aliasing cuasing weird space, also centering logo on mobile. (#2187)

Commit:
fix(urls): Use relative URLs instead of linking to docs.sentry.io (#2382)

* fix(urls): Use relative URLs instead of linking to docs.sentry.io

* Revert relative URL change to api docs link

* Fix grouping & fingerprinting link

* Fix broken url on log4net docs

MimiDumpling committed:
fixed broken anchor tags (#2393)

maciejwalkowiak committed:
Update Java SDK docs to 3.0 (#2239)

Co-authored-by: Fiona <61481573+PeloWriter@users.noreply.github.com>
Co-authored-by: Bruno Garcia <bruno@brunogarcia.com>
Co-authored-by: Manoel Aranda Neto <5731772+marandaneto@users.noreply.github.com>

bruno-garcia committed:
Android: Current migration on top (#2396)

tonyo committed:
relay: Next iteration of refactoring (#2365)

Commit:
version update

dcramer committed:
feat: Add example of how to sample rrweb (#2398)

Co-authored-by: Fiona <61481573+PeloWriter@users.noreply.github.com>

dcramer committed:
feat: Improve Product Guides structure (#2399)

- Expose dropdown for Product navigation
- Move all product pages into /product/
- Re-order sidebar to push relevant (vs marketing/discovery) docs to the top

lobsterkatie committed:
feat(performance): Add `tracesSampleRate` and `tracesSampler` to JS docs (#2376)

Also a bunch of reorganization, and reworking of some (but not all) of the JS performance docs.

dcramer committed:
fix: Language

ceorourke committed:
feat(api-docs): Remove old unused API docs files (#2400)

Commit:
version fix for GCP

untitaker commented:
@ajjindal the rebase is messed up, this PR now contains too many unrelated changes to be reviewable
